docs/0.3-DRAFT/index.md

```diff
@@ -1324,7 +1276,7 @@ To record curation actions which modify a [File](http://schema.org/MediaObject)
 ```json
 {
     "@id": "#history-01",
-    "@type": "CreateAction",
+    "@type": "#CreateAction",
```

Comment:
No `#`, those types are schema.org terms from the `@context`, e.g. <https://schema.org/CreateAction>

Still there COULD be other types of custom actions like `#ImagineAction`, but perhaps we should not go into that.